db/db_impl/db_impl.cc

```diff
@@ -2301,7 +2301,8 @@ Status DBImpl::CreateColumnFamilyImpl(const ColumnFamilyOptions& cf_options,
       auto* cfd =
           versions_->GetColumnFamilySet()->GetColumnFamily(column_family_name);
       assert(cfd != nullptr);
-      s = cfd->AddDirectories();
+      std::map<std::string, std::shared_ptr<Directory>> dummy_created_dirs;
+      s = cfd->AddDirectories(&dummy_created_dirs);
```

Comment:
Are we passing an empty map here?

Reply:
Yes.

Reply:
It seems to me that the new column family's cf_paths can overlap with existing cf_paths. Should we populate dummy_created_dirs with existing cf_paths to avoid creating dir and Directory for them?

Reply:
We could, but it would be a little bit complex and I don't want to be so complicated.